### PR TITLE
[Validator] 19 digits VISA card numbers are valid

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 2.7
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        | 

some sources:

* https://en.wikipedia.org/wiki/Payment_card_number
* https://blog.vindicia.com/19-digit-visas-and-2-digit-mastercards/
* https://www.nccgroup.trust/de/uber-uns/newsroom-and-events/blogs/2016/november/prepare-for-19-digit-credit-cards/
* http://support.worldpay.com/support/kb/bg/testandgolive/tgl5103.html

### CODE REVIEW

**Review by nicolas-grekas — APPROVED** [review]

File: src/Symfony/Component/Validator/Constraints/CardSchemeValidator.php [hunk 1]
```diff
@@ -81,7 +81,7 @@ class CardSchemeValidator extends ConstraintValidator
         ),
         // All Visa card numbers start with a 4. New cards have 16 digits. Old cards have 13.
```
Comment (dbrumann) [inline src/Symfony/Component/Validator/Constraints/CardSchemeValidator.php]: Comment should be removed or updated as well?

**Review by javiereguiluz — APPROVED** [review]
@xabbuh thank you for fixing this error. Please, don't forget to update the comment too. Thanks!

**Review by Simperfit — APPROVED** [review]

File: src/Symfony/Component/Validator/Constraints/CardSchemeValidator.php (hunk 1 above)
Comment (xabbuh) [reply src/Symfony/Component/Validator/Constraints/CardSchemeValidator.php]: comment updated

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**